actions/team_actions.jsx

```diff
@@ -43,10 +43,10 @@ export async function updateTeam(team, onSuccess, onError) {
 }
 
 export async function removeUserFromTeam(teamId, userId, success, error) {
-    const {data, error: err} = await TeamActions.removeUserFromTeam(teamId, userId)(dispatch, getState);
-    getUser(userId)(dispatch, getState);
-    TeamActions.getTeamStats(teamId)(dispatch, getState);
-    getChannelStats(ChannelStore.getCurrentId())(dispatch, getState);
+    const {data, error: err} = await dispatch(TeamActions.removeUserFromTeam(teamId, userId));
```

Comment:
@jwilander  removeUserFromTeam needs getState. Would this work without that?

Reply:
It will actually get passed `getState` automatically by the dispatch, there's no need for us to pass it manually. We do it all over the code because we didn't know that at first